Makefile

```diff
@@ -958,8 +958,10 @@ hint: node_modules/.bin/jshint
 	@./node_modules/.bin/jshint $(JSHINT_ARGS) $(JSHINTED_PATH) $(LINTED_FILES) || (echo Please consult https://github.com/mozilla-b2g/gaia/tree/master/build/jshint/README.md to get some information about how to fix jshint issues. && exit 1)
 
 eslint: node_modules/.bin/eslint
+	sed 's/\s*#.*$$//' build/eslint/xfail.list | uniq > build/eslint/xfail.tmp
```

Comment:
please add the tempoarary file to .gitignore
the goal is that if eslint is failing, the temporary files does not stay in the middle:)